<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Zakaz</class>
 <widget class="QWidget" name="Zakaz">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>370</width>
    <height>362</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background: rgb(24, 24, 24)</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>20</y>
     <width>141</width>
     <height>16</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel {color: rgb(85, 255, 0);
			 font: bold;			}</string>
   </property>
   <property name="text">
    <string>Выберите ресторан:</string>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBox">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>50</y>
     <width>231</width>
     <height>22</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QComboBox {color: rgb(85, 255, 0);
			 font: bold;			}</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>240</y>
     <width>182</width>
     <height>23</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {background-color: rgb(255, 255, 0);
     border-style: outset;
     border-radius: 10px;
     border-color: beige;
     font: bold 14px;
     min-width: 10em;
     padding: 6px;}
QPushButton::hover{background-color: rgb(217, 217, 0)}
QPushButton::pressed{background-color: rgb(181, 181, 0)}</string>
   </property>
   <property name="text">
    <string>Заказать</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>90</y>
     <width>121</width>
     <height>16</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel {color: rgb(85, 255, 0);
			 font: bold;			}</string>
   </property>
   <property name="text">
    <string>Выберите блюдо:</string>
   </property>
  </widget>
  <widget class="QPlainTextEdit" name="plainTextEdit">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>270</y>
     <width>191</width>
     <height>71</height>
    </rect>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
   <property name="plainText">
    <string>Информация</string>
   </property>
  </widget>
  <widget class="QTableWidget" name="tableWidget">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>120</y>
     <width>321</width>
     <height>111</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QTableWidget {color: rgb(85, 255, 0);
			 font: bold;			}</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
